html/changelog.html

```diff
@@ -1799,6 +1799,16 @@ <h3 class="author">tralezab updated:</h3>
 			<ul class="changes bgimages16">
 				<li class="rscadd">The Psychology Office now has a small positive moodlet, and some ambience</li>
 			</ul>
+
+			<h2 class="date">24 March 2020</h2>
+			<h3 class="author">ArcaneDefence updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">Vapes loaded with welding fuel no longer prevent you from being further set on fire from other sources.</li>
+			</ul>
+			<h3 class="author">bobbahbrown updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">Fixed runtime on donutstation caused by mistype in dmm file</li>
+			</ul>

```

Comment:
Unintended changes

Reply:
i literally replaced changelog file on this pr with the one found on master.